pkg/net/http/httputils.go

```diff
@@ -273,6 +276,7 @@ func (qh *QueryHeader) ToOffsetPagination() Pagination {
 		SortOrder: qh.SortOrder,
 		StartDate: qh.StartDate,
 		EndDate:   qh.EndDate,
+		Alias:     qh.Alias,
```

Comment:
Validate if alias should be +1
```suggestion
		Aliases:     qh.Aliases,
```